actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -175,6 +186,89 @@ private void updateUser(Request actorMessage) {
     TelemetryUtil.telemetryProcessingCall(userMap, targetObject, correlatedObject);
   }
 
+  private void validateUserOrganisations(Request actorMessage) {
+    if (null != actorMessage.getRequest().get(JsonKey.ORGANISATIONS)) {
+      List<Map<String, Object>> userOrgList =
+          (List<Map<String, Object>>) actorMessage.getRequest().get(JsonKey.ORGANISATIONS);
+      for (Map<String, Object> userOrg : userOrgList) {
+        String orgId = (String) userOrg.get(JsonKey.ORGANISATION_ID);
+        Organisation organisation = organisationClient.esGetOrgById(orgId);
```

Comment:
use esGetOrgbyIds -> so all org ids will be loaded - and throw error for missing organisations together